plugins/openshift/commands/generate-api.md

```
@@ -0,0 +1,116 @@
+---
+description: Generate OpenShift API stubs with automatic error detection and fixing, optionally updating client-go
+argument-hint: "[api-repo-path] [--client-go [client-go-path]]"
+---
+
+## Name
+openshift:generate-api
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)